ledger/src/shred/merkle.rs

```diff
@@ -190,9 +190,24 @@ impl ShredData {
         else {
             return Err(Error::InvalidShredVariant);
         };
+        Self::get_chained_merkle_root_offset(proof_size, resigned)
+    }
+
+    pub(crate) fn get_chained_merkle_root_offset(
```

Comment:
shouldn't these be `pub(super)` as well?
similarly the other one below?

Reply:
ah ya good call, still get confused on how module hierarchy works sometimes.

`shred.rs` declares `mod merkle`, and the `mod layout` is able to access `ShredData/ShredCode::get_merkle_root` as a parent module? 😕 . i would've assumed they are siblings.
```
              /- shred_code
              /- shred_data
ledger - shred - layout
```